common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,220 @@
+#### Local Types ####
```

Comment:
Works better as the iron start art IMHO.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105626080-1997cd80-5e47-11eb-8524-7932e69208bc.png)

</details>